### DISCUSSION TITLE
如何预设翻译语言

### DISCUSSION THREAD

**Discussion by czjabc** [YiiGuxing/TranslationPlugin#1681] [Q&A] [answered]
我想把英文翻译成中文，但每次调起插件，都是“English->English”的模式，需要手动切到Chinese语言，特别不方便。想问一下，有没什么方式可以预设翻译前后的语言，不用每次手动改

**Comment by YiiGuxing** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10737066/176817582-956082f6-1dd2-42c4-a21a-cfa16d06ba12.png)

**Reply by czjabc** [reply]
啊，原来设置在这里，可以快乐翻译了，万分感谢！

**Reply by SakurajimaMaii** [reply]
好家伙，原来在这里
